tests/end_to_end/end_to_end.cpp

```diff
@@ -924,6 +925,7 @@ _cgroup_load_test(
     bpf_object__close(object);
 }
 
+#if defined(_DEBUG)
```

Comment:
Why is this test under `#if defined(_DEBUG)`?

Reply:
Because this program type fails to verify due to an unknown program type.

Reply:
Why is the verification failing? We already have these tests passing for .o files below, so verification should already be succeeding?

Reply:
https://github.com/microsoft/ebpf-for-windows/blob/b01dea6d55cf31c38d9e464c4dc80ebae2396c93/tests/end_to_end/end_to_end.cpp#L953

Reply:
Verification relies on either offline program information or querying it from the program information provider via kernel EC. bpf2c running in CI/CD doesn't have access to the kernel EC and this program type isn't present in the offline program information.

Reply:
Then it should be failing in the current end-to-end tests also?

Reply:
This assert fires:
https://github.com/microsoft/ebpf-for-windows/blob/bd2aa504e1b97f40e99fc35f796e0c9fc0878100/libs/api_common/windows_helpers.cpp#L133

Reply:
https://github.com/microsoft/ebpf-for-windows/issues/971

Reply:
Should we add a comment in the code with TODO# bug number?